static/fetchMovieScreenings.js

```diff
@@ -0,0 +1,8 @@
+export default async function fetchMovieScreenings() {
+    const currentPath = window.location.pathname
+    const base = window.location.origin;
+    const url = new URL("/api" + currentPath + "/screenings", base)
```

Comment:
You shouldn't have to worry about `base`. If you use a URL that starts with `/`, the browser will interpret it as being on the same host (domain/hostname and port) as the page on which the code is executing. So if the code is executing on localhost:5080/some/page, and requests /api/movies/1/screenings, the browser will interpret that as localhost:5080/api/movies/1/screenings and make the request to there.